cli/src/alluxio.org/cli/cmd/generate/template.go

```diff
@@ -0,0 +1,62 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package generate
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"alluxio.org/cli/env"
+	"alluxio.org/log"
+)
+
+var Template = &TemplateCommand{}
+
+type TemplateCommand struct {
+	conf string
+}
+
+func (c *TemplateCommand) ToCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "template --conf",
+		Short: "Generate a config file if one doesn't exist.",
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) == 0 {
+				log.Logger.Infoln("Usage: template <alluxio_master_hostname>")
+				return
+			}
```

Comment:
```suggestion
```
remove this since it will be checked by `Args:  cobra.ExactArgs(1),` and the argument description is part of the `Use`